jbpm-wb-executor-service/jbpm-wb-executor-service-client/src/main/java/org/jbpm/workbench/es/client/editors/requestlist/RequestListPresenter.java

```diff
@@ -254,26 +254,32 @@ public void setupAdvancedSearchView() {
 
         view.addTextFilter(constants.BusinessKey(),
                            constants.FilterByBusinessKey(),
-                           v -> addAdvancedSearchFilter(equalsTo(COLUMN_BUSINESSKEY,
-                                                                 v)),
-                           v -> removeAdvancedSearchFilter(equalsTo(COLUMN_BUSINESSKEY,
-                                                                    v))
+                           v -> addAdvancedSearchFilter(likeTo(COLUMN_BUSINESSKEY,
```

Comment:
@cristianonicolai  I think we have to add "%" + v'+ "%" to do the likeTo really useful and different from the equalsTo